packages/aragon-api/src/index.js

```diff
@@ -72,6 +72,23 @@ export class AppProxy {
     )
   }
 
+
+  /**
+   * Check user membership.
+   *
+   * Membership is determined by checking any token balance in all
+   * of the applications Token Manager applications.
+   *
+   * @param  {string} address The address of the account to check
+   * @return {void}
+   */
+  isMember (address) {
+    this.rpc.send(
```

Comment:
I don't think this will return a value unless it's a `sendAndObserveResponse` call. You'll also probably want to verify that a stubbed response is returned as well.